src/state/action-creators.js

```diff
@@ -69,7 +69,6 @@ export type CompleteLiftAction = {|
     id: DraggableId,
     type: TypeId,
     client: InitialDragLocation,
-    page: InitialDragLocation,
```

Comment:
Removed page from all these actions - easy to compute once with windowScroll in the reducer